blog/views.py

```diff
@@ -0,0 +1,64 @@
+from django.shortcuts import HttpResponseRedirect
+from django.urls import reverse_lazy, reverse
+from django.views import generic
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic.edit import FormMixin
+from .models import Post
+from .forms import CommentaryForm, PostForm
+from django import forms
+
+
+class Index(generic.ListView):
+    model = Post
+    queryset = Post.objects.order_by("-created_time")
```

Comment:
better practice is to define the default ordering for a model within the model's Meta class rather than specifying it in the view. This helps keep your code organized and follows the DRY